### PR TITLE
Adding page for istio image signing and validation

### PR DESCRIPTION
**Pull request by stewartbutler**
Adds documentation for image signing and validation.

I'd appreciate discussion of where this should go. The logical place for it is in best-practices/security/index.md, but that is getting a bit long in the tooth. I created it as a separate page for now, but it may be better to split up the security page into a collection of pages.

We may also want a blurb somewhere in the installation docs referencing this, but I'm not sure where would be best.

Holding merge with do-not-merge label until we have settled on something.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [X] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (jacob-delgado) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:17]:
```suggestion
simplifies signing and validation of signed Open Container Initiative (OCI) artifacts,
```

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (jacob-delgado) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:21]:
```suggestion
process using our own private key starting with Istio 1.12. End users can then verify these images using
```

File: static/misc/istio-key.pub
Comment (jacob-delgado) [inline static/misc/istio-key.pub:1]: put a CODEOWNERS in this directory for either T&R or TOC or Environment

**Review by jacob-delgado — APPROVED** [review]
lgtm, just needs codeowners

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (ericvn) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:6]: Not sure we need an alias for a new page.

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (ericvn) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:7]: Do we want some more keywords, like `signing`? Not sure what would make sense.

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (ericvn) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:8]: Should environments own this page?

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (ericvn) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:9]: Might be interesting to actually add a test for this page if we don't actively check that the commands work elsewhere.

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (ericvn) [reply content/en/docs/ops/best-practices/image-signing-validation/index.md:21]: We may actually be able to say 1.11.x (not sure what x is).

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (ericvn) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:51]: with the

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (ericvn) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:48]: The pub key isn't working correctly, at least if I look at the deploy/netlify test details link, https://deploy-preview-10775--preliminary-istio.netlify.app/latest/docs/ops/best-practices/image-signing-validation/

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (stewartbutler) [reply content/en/docs/ops/best-practices/image-signing-validation/index.md:8]: I wasn't sure, here. AFAIK no existing pages are owned by test and release, and it wasn't clear if this was a group that was internal to the istio.io repo or if it was referencing github org groups.

Comment (ericvn) [reply content/en/docs/ops/best-practices/image-signing-validation/index.md:8]: It would refer to the WG that would approve changes to this page (or be responsible if a test suddenly started failing (noting no test on this page so can ignore that)). A doc reviewer, may not understand the technicalities of the page so would expect that someone from the specified WG would approve the PR before it would be approved by the doc reviewer and merge.

Comment (jacob-delgado) [reply content/en/docs/ops/best-practices/image-signing-validation/index.md:8]: The environments group will take ownership of this page. That's fine with me.

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (stewartbutler) [reply content/en/docs/ops/best-practices/image-signing-validation/index.md]: Done.

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (stewartbutler) [reply content/en/docs/ops/best-practices/image-signing-validation/index.md:47]: It looks like it is setting `$.Site.Page.BaseURL` to `/latest/` rather than the full URL.

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (stewartbutler) [reply content/en/docs/ops/best-practices/image-signing-validation/index.md]: Removed

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (stewartbutler) [reply content/en/docs/ops/best-practices/image-signing-validation/index.md]: Added one, but also not sure what else makes sense.

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (stewartbutler) [reply content/en/docs/ops/best-practices/image-signing-validation/index.md:7]: Leaving as Environments, then.

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (sabre1041) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:37]:
```suggestion
    -verify <(curl -ssL https://raw.githubusercontent.com/sigstore/cosign/main/release/release-cosign.pub) \
```

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (sabre1041) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:38]:
```suggestion
    -signature <(cat /path/to/cosign.sig | base64 -d) \
```

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (sabre1041) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:46]:
```suggestion
To validate a container image, do the following:
```
Suggest using "container image" throughout document

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (sabre1041) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:42]:
```suggestion
1. Make the binary executable (`chmod +x`) and move to a location on the `PATH`
```

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (sabre1041) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:11]:
```suggestion
This page describes how to use [cosign](https://github.com/sigstore/cosign) to
```

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (sabre1041) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:14]:
```suggestion
cosign is a tool developed as part of the
```

**Review by sabre1041 — CHANGES_REQUESTED** [review]

**Comment by kfaseela** [comment]
Thanks @sabre1041 for the review!

**Comment by ericvn** [comment]
We should try and get this in before 1.14 is released.

**Comment by stewartbutler** [comment]
@ericvn Accepted the suggestions and I think it is ready to go now. Just needs @sabre1041 to accept the CLA.

**Comment by sabre1041** [comment]
@stewartbutler accepted CLA

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (craigbox) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:2]:
```suggestion
title: Image Signing and Validation
```

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (craigbox) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:11]:
```suggestion
This page describes how to use [Cosign](https://github.com/sigstore/cosign) to
```

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (craigbox) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:14]:
```suggestion
Cosign is a tool developed as part of the
```

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (craigbox) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:17]:
```suggestion
[sigstore](https://www.sigstore.dev) project, which
simplifies signing and validation of signed Open Container Initiative (OCI) artifacts,
such as container images.
```

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (craigbox) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:20]:
```suggestion
Starting with Istio 1.12, we sign all officially published container images as part of our release
process. End users can then verify these images using
```

File: content/en/docs/ops/best-practices/image-signing-validation/index.md
Comment (craigbox) [inline content/en/docs/ops/best-practices/image-signing-validation/index.md:33]:
```suggestion
1. Validate the `cosign` binary signature:
```

**Review by craigbox — COMMENTED** [review]
copy editing

**Comment by stewartbutler** [comment]
@craigbox Addressed, please take another look?

**Comment by stewartbutler** [comment]
@ericvn Leaving the do-not-merge label on, feel free to remove it when you are happy with it.

**Review by ericvn — APPROVED** [review]
Thanks Stewart

**Pull request merged**
